config.py

```diff
@@ -0,0 +1,58 @@
+# config.py
+
+import constants
+import datetime
+import json
+import jwt
+import requests
+from pathlib import Path
+
+
+def get_encoded_jwt_token(credentials: dict) -> str:
```

Comment:
Should we pass the whole credentials object or only the techid and orgid ?

Reply:
In this function I am using 4/6 keys of the credentials dictionary. So is it a better practice to pass only those keys separately that are going to be used in this method?